docs/editors/vim.md

```diff
@@ -411,6 +411,14 @@ install [coc-json](https://github.com/neoclide/coc-json).
 ```scala mdoc:user-config:lsp-config-coc
 ```
 
+```scala mdoc:parent-lenses:vim
```

Comment:
I actually have a whole section on this further down. https://scalameta.org/metals/docs/editors/vim.html#goto-super-method

I'd say remove it and replace it with this, but then my readme and this document becomes quite a bit different. I don't love having them both, but I also don't like not having a detailed readme 🤔 I'm not really sure what would be best for this.

Reply:
Haven't noticed it. I can just add it to the VS Code section then.

Reply:
Removed from vim section.